src/content/docs/infrastructure/host-integrations/host-integrations-list/apache-druid-integration.mdx

```diff
@@ -0,0 +1,266 @@
+---
+title: Apache Druid integration
+tags:
+  - Druid integration
+  - apache druid integration
+  - New Relic integrations
+metaDescription: Use New Relic infrastructure agent to get a dashboard with druid metrics.
+freshnessValidatedDate: 2024-04-21
+---
+
+import infrastructureDruidDashboard from 'images/infrastructure_screenshot-full_apache-druid-dashboard.webp'
+
+Integrating Apache Druid with New Relic enhances your ability to monitor, analyze, and optimize the performance of your Druid clusters. By leveraging New Relic's powerful monitoring and alerting features, you can ensure the reliability and efficiency of your Druid-based analytics platform.
```

Comment:
"enhances" is a verb that i need to weave in sometimes...

Reply:
https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#New-Relic-brand-name 

We aren't supposed to make New Relic possessive unfortunately... maybe to simplify the language, I'd just say "This integration gives you a monitoring tool with alerting features, ensuring the reliability and blahblahblah"